collect_app/src/androidTest/java/org/odk/collect/android/support/AdbFormLoadingUtils.java

```diff
@@ -58,10 +58,6 @@ public static void copyFormToStorage(String formFilename, List<String> mediaFile
         }
     }
 
-    public static void copyFormToStorage(String formFilename, String projectName) throws IOException {
```

Comment:
I think you can convert this class to kotlin too.

Reply:
Converted.